src/main/java/fi/csc/chipster/jobhistory/JobHistoryModel.java

```diff
@@ -41,6 +41,7 @@
 	private String timeDuration;
 	private Instant startTime;
 	private Instant endTime;
+	private String userName;
```

Comment:
The username is written as single word elsewhere, so no capital N. Change the method names too